tina/collections/articles.tsx

```diff
@@ -0,0 +1,317 @@
+            },
+            {
+              value: "France",
+              label: "France",
+            },
+            {
+              value: "China",
+              label: "China",
+            },
+          ],
+        },
+        {
+          type: "rich-text",
+          label: "Description",
+          name: "description",
+        },
+      ],
+    },
+  ],
+};
+
+export const articlesIndexSchema: Collection = {
+  label: "Articles - Index",
+  name: articlesIndexSchemaConstants.value,
+  format: "mdx",
+  path: "content/articles/index",
+  ui: {
+    router: () => {
+      return "/articles";
+    },
+    allowedActions: {
+      create: false,
+      delete: false,
+    },
+  },
+  fields: [
+    {
+      type: "object",
+      label: "Header Image",
+      name: articlesIndexSchemaConstants.headerImage.value,
+      fields: [
+        {
+          type: "image",
+          label: "Hero Background",
+          name: articlesIndexSchemaConstants.headerImage.heroBackground,
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore
+          uploadDir: () => "background",
+        },
+        {
+          type: "string",
+          label: "Alt Text",
+          name: articlesIndexSchemaConstants.headerImage.altText,
+        },
+        {
+          type: "string",
+          label: "Text Overlay",
+          name: articlesIndexSchemaConstants.headerImage.txtOverlay,
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore
+          uploadDir: () => "background",
+        },
+      ],
+    },
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    seoSchema,
+    {
+      type: "string",
+      label: "Title",
+      name: articlesIndexSchemaConstants.title,
+    },
+    {
+      type: "rich-text",
+      label: "Body",
+      name: articlesIndexSchemaConstants._body,
+      templates: [...Schemas.pageBlocks],
+      isBody: true,
+    },
+    {
+      type: "boolean",
+      name: "fullWidthBody",
+      label: "Full Width Body",
+    },
+    {
+      type: "boolean",
+      name: "showSidebarPanel",
+      label: "Show Sidebar Panel",
+      required: false,
+    },
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    sidebarPanelSchema,
+    {
+      type: "object",
+      label: "Articles",
+      description: "Articles",
+      name: articlesIndexSchemaConstants.articles.value,
+      ui: {
+        itemProps: (item) => {
+          return { label: item?.title };
+        },
+      },
+      list: true,
+      fields: [
+        {
+          type: "string",
+          label: "Title",
+          name: articlesIndexSchemaConstants.articles.title,
+        },
+        {
+          type: "rich-text",
+          label: "Body",
+          name: articlesIndexSchemaConstants.articles.body,
+        },
+        {
+          type: "string",
+          label: "Page URL",
```

Comment:
How does this work? Shouldn’t the article title form part of the url?

Reply:
I'm not seeing this prop used anywhere so going to remove it

Reply:
Done - I've removed the prop because it's not used